README.md

```diff
@@ -132,6 +132,10 @@ selectively enabled or disabled:
   true if cljfmt should collapse consecutive blank lines. This will
   convert `(foo)\n\n\n(bar)` to `(foo)\n\n(bar)`. Defaults to true.
 
+* `:split-keypairs-over-multiple-lines?` -
+  true if cljfmt should break hashmaps onto multiple lines.
```

Comment:
Lines in the README should be wrapped at 72 characters.

Reply:
Wrapping is still incorrect. It should look like:

```
  true if cljfmt should break hashmaps onto multiple lines. This will
  convert `{:a 1 :b 2}` to `{:a 1\n:b 2}`. Defaults to false.
```